http/vibe/http/proxy.d

```diff
@@ -226,6 +260,8 @@ final class HTTPReverseProxySettings {
 	/// ditto
 	@property void destinationPort(ushort port) { destination.port = port; }
 
+	/// Whether the proxy should act as forward or reverse proxy
+	bool reverseProxyMode = false;
```

Comment:
Again, to not break existing code that uses the first `listenHTTPReverseProxy` overload, the default would have to be `true` here.